mw-webapp/src/model/firebaseCollection/StudentComment.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Student comments
+ */
+export class StudentComment {
+
+  /**
+   * Comment's text
+   */
+  public description: string;
+
+  /**
+   * True if comment was done and false if not
+   */
+  public isDone: boolean;
+
+  constructor(studentCommentData: StudentComment) {
+    this.description = studentCommentData.description;
+    this.isDone = studentCommentData.isDone;
+  }
```

Comment:
So does we need a option to mark simple comments as done or not? I'm just refactoring the existing functionality

Reply:
But this cells will always contain simple comments

Reply:
So we don't need a option to mark it as done \ completed?

Reply:
Give please example when studentComment can be with property isDone: false

Reply:
Maybe i'm not understanding the question correctly but in this context studentComment should have isDone: false by default

Reply:
Why it should be false by default? 
Give example of real student comment. (I mean image that student wrote it in the cell)/ So give some examples. what the student can write in this sell with property  isDone: false

For example if student will write ""It was hard to read 100 pages of MsConel" it can't be false

Reply:
Fully deleted this model